components/actionbutton/themes/spectrum.css

```diff
@@ -21,10 +21,19 @@
 		--spectrum-actionbutton-background-color-hover: var(--spectrum-gray-200);
 		--spectrum-actionbutton-background-color-down: var(--spectrum-gray-300);
 		--spectrum-actionbutton-background-color-focus: var(--spectrum-gray-200);
-		--spectrum-actionbutton-background-color-selected-disabled: var(--spectrum-transparent-black-200);
+		--spectrum-actionbutton-background-color-disabled: var(--spectrum-gray-75);
```

Comment:
I'm wondering about this one... digging for the [S1 spec](https://adobe.sharepoint.com/sites/Spectrum/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FSpectrum%2FShared%20Documents%2FSpectrum%20Design%2FToken%20specs&csf=1&web=1&e=9M0cBH&CID=1a749a26%2De833%2D4461%2Da227%2D5cdce87deb5f&FolderCTID=0x0120004AB1576A94E3914C949954710369C339) and also looking at the last [PR preview](https://pr-3524--spectrum-css.netlify.app/?path=/story/components-action-button--default&globals=testingPreview:!true) before Foundations was merged, it looks like we might want `transparent` for this instead? Unless there's a particular reason for this to be `gray-75` instead?

Reply:
It looks like the changes resolved the initial issue (missing background for "Disabled + Selected" in Dark theme), but maybe need a second pass to keep transparent the default for `disabled`, and the gray the default for `selected-disabled` and then only do variant overrides when it changes to simplify and keep the naming meaningful.

Reply:
ohohoho good calls all around. I was thinking just to keep the S1 background color I saw on prod, but I see now that the S1 background color is actually incorrect in the first place. Sorry about the misunderstanding! I'll see if I can dig into these once more.

Reply:
Ok I think I've the transparent background back for disabled S1 buttons. https://github.com/adobe/spectrum-css/pull/3540/commits/fb4cc7b8fbbc28e26a87cf6e761a9c02a83276b4 Let me know if something still looks off!

<img width="1207" alt="Screenshot 2025-02-11 at 5 05 52 PM" src="https://github.com/user-attachments/assets/7b748ac6-e0ea-4d7d-8ccd-85ca85dcefe0" />